src/crimson/os/seastore/onode_manager/staged-fltree/node.cc

```diff
@@ -747,7 +747,9 @@ eagain_future<> Node::retire(context_t c, Ref<Node>&& this_ref)
   assert(this_ref->use_count() == 1);
 
   return impl->retire_extent(c
-  ).safe_then([this_ref = std::move(this_ref)]{ /* deallocate node */});
+  ).safe_then([this_ref = std::move(this_ref)] () mutable {
```

Comment:
i don't follow on this. why `mutable` matters in this context?

Reply:
I also don't understand it.  1, why we need transfer this_ref, each Node has a this pointer,  we only need capture this pointer is OK, 2. why mutable here, seems nothing to be changed in the continuation. Do you mean this_ref is released here? without mutable, this_ref can't be released?

Reply:
removed, this mutable is not needed.